cds/modules/theme/static/js/cds/module.js

```diff
@@ -212,20 +212,65 @@ app.filter('findResolution', function($filter) {
   return function(record) {
     height = record['tags']['height'];
     selectedResolution = height.concat('p');
+
+    configuredResolutionsWidth = {
+      '1920': '1080p',
+      '2048': '2K'
+    }
+
+    configuredResolutions = {
+      '240': '240p',
+      '360': '360p',
+      '480': '480p',
+      '720': '720p',
+      '1024': '1024p',
+      '1080': '1080p',
```

Comment:
double key in the object. Also why you mapped 1080 to 2K, 2160 to 4K...I mean why the round up?

Reply:
So this is the wrong filter I deleted it. :) The next one is the correct one. And yes we map 1080 to 2K verify the width. Because 2K has height of 1080.